content/2. Ondersteunende-informatie/4. Techniek/Git/6. Git files.md

```diff
@@ -0,0 +1,88 @@
+---
+title: 6. Git files
+draft: false
+---
+
+Met Git zijn er sommige documenten of folders wat Git herkent en kan gebruiken voor eigen acties. 
+## .git/
+De `.git` folder is een verborgen map in de root van een project. De map bevat alle metadata en objecten die Git nodig heeft om de repository te herkennen en beheren zoals de commitgeschiedenis, configuraties, branches, tags en stashes. Verder zijn er ook nog folders in de `.git` folder die bepaalde taken hebben
+- `objects/`: Bevat alle objecten die Git nodig heeft voor het versiebeheer (zoals commits).
+- `refs/`: Bevat verwijzingen naar branches en tags.
+- `config`: De repository-specifieke configuratie.
+- `HEAD`: Bevat een verwijzing naar de huidige actieve branch.
+### .git/hooks/
+De `hooks` subfolder bevat scripts die automatisch worden uitgevoerd door Git bij bepaalde gebeurtenissen of acties in de levenscyclus van een repository. Deze gebeurtenissen omvatten zaken zoals commits, merges, pushes, en meer. 
+
+Git hooks geven je de mogelijkheid om taken te automatiseren, zoals het valideren van code voordat je deze commit, het versturen van meldingen na een push, of het controleren van codeformattering. Een paar voorbeelden van zo'n hook is een `pre-commit`, `post-commit` en `pre-push`. 
+
+> Meer informatie over git hooks: https://git-scm.com/book/ms/v2/Customizing-Git-Git-Hooks
+
+## .gitignore
```

Comment:
Handig te benoemen dat je als eerst een gitignore moete maken. zodra een bestand is gecommit is het een hoop gedoe om het weer aan te passen